src/components/MoneyRequestHeader.js

```diff
@@ -90,6 +90,11 @@ function MoneyRequestHeader(props) {
     const shouldShowSettlementButton = !isSettled && !props.isSingleTransactionView && isPayer;
     const bankAccountRoute = ReportUtils.getBankAccountRoute(props.chatReport);
     const shouldShowPaypal = Boolean(lodashGet(props.personalDetails, [moneyRequestReport.managerID, 'payPalMeAddress']));
+    const report = props.report;
+    if (props.isSingleTransactionView) {
+        report.ownerAccountID = lodashGet(props, ['parentReport', 'ownerAccountID'], null);
+        report.ownerEmail = lodashGet(props, ['parentReport', 'ownerEmail'], '');
+    }
```

Comment:
@luacmartins, can you check if this change and the one below where i remove parentReport from HeaderWithBackButton breaks anything with single expense/IOU view?

also cc @grgia for awareness as this code change affects report headers since we're providing source report instead of parent report to HeaderWithBackButton.

Reply:
Some of [these checks](https://github.com/Expensify/App/blob/38b095fdcda09ba3dabfba1da8052441a64ad93d/src/components/AvatarWithDisplayName.js#L57C1-L59) depend on the report type, which can be different with your changes. Previously, the report type of the parentReport on a singleTransactionView was `iou` or `expense` and now it'll always be `chat`.

The only difference that I noticed with your changes is that the Workspace subscript avatar is missing in the single transaction view. There might be others given the type checks we do in the code I linked above.

**Before**
![expense-transaction](https://github.com/Expensify/App/assets/22219519/9a54836d-b00a-41b4-bbba-b73bb96b4aa6)

**After**
![Screenshot 2023-06-21 at 1 09 39 PM](https://github.com/Expensify/App/assets/22219519/a60bb684-99f1-48e6-b9f6-df08a272cfdc)

Reply:
Hmm i think that geogia is changing the avatar anyway. @grgia do you know what the final avatar for this single expense request is supposed to be. Is it supposed to be with a workspace subscript? If so i think i can easily add it in using the current report instead of having the parent report.